components/dashboard/src/components/Modal.tsx

```diff
@@ -4,7 +4,8 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-import { useEffect } from "react";
+import { ReactNode, useEffect } from "react";
+import cn from "classnames";
```

Comment:
That makes sense. Would you make a drive-by fix the [package.json](https://github.com/gitpod-io/gitpod/blob/8dd290cf0be4baa2cd6df5e87729f78c782e9722/components/dashboard/package.json#L46) to make it a runtime dependency, please? 🙏🏻

Reply:
moved it from devDeps to normal deps - is that what you were referring to? Thanks for catching that, I didn't realize it was just in devDeps)